File: 백준/Bronze/11094. 꿍 가라사대/꿍 가라사대.py
import sys
input = lambda: sys.stdin.readline().rstrip()


def main():
    N = int(input())
    for _ in range(N):
        S = input()
        if S[:10] == 'Simon says':
            print(S[10:])


if __name__ == '__main__':
    main()

File: 백준/Bronze/11094. 꿍 가라사대/꿍 가라사대.c
#include <stdio.h>
#include <string.h>

int main() {
    int N;
    scanf("%d\n", &N);
    char S[102];  
    for (int i = 0; i < N; i++) {
        fgets(S, sizeof(S), stdin);
        if (strncmp(S, "Simon says", 10) == 0) {
            printf("%s", S + 10);
        }
    }
    return 0;
}